#57 - Remove Scraper and Display RMP Ratings

dvegas03 opened the pull request:
- Remove Python Scraper
- Update insert.ts (avgRating db column)
- Update schema.ts (avgRating db column)
- Update Course page (actual avgRating display instead of placeholder)
- Warning: DB schema changes!
Closes #57

dvegas03 committed:
#57 - Remove Scraper and Display RMP Ratings

- Remove Python Scraper
- Update insert.ts (avgRating db column)
- Update schema.ts (avgRating db column)
- Update Course page (actual avgRating display instead of placeholder)
- Warning: DB schema changes!
Closes #57

YuraT reviewed (changes requested):
when changing the schema, commit the generated migrations after running `db:generate`

joshua-demo commented on src/app/courses/[id]/page.tsx:
```diff
@@ -59,7 +59,7 @@ export default async function CoursePage({
                       <div className="mt-2">
                         {/* The actual rating is a placeholder, the professorsTable scheme has not been updated to have starRating as a field */}
-                        <StarRating rating={4.5} textColor="text-muted-foreground" />
+                        <StarRating rating={result.professor.avgRating ?? 0} textColor="text-muted-foreground" />
```
Instead of saying 0 for a rating, put the text "no rating" or some other error message

dvegas03 committed:
Handle missing professors + db migrations

- Add db migrations after adding "avg_rating" column
- Add fallback to "No rating" after "avg_rating" is null/0 on the frontend
Closes #57

dvegas03 commented:
@YuraT Pushed updated commits addressing feedback, ready for re-review. Thanks!

YuraT reviewed (approved):
:star: looks good

joshua-demo reviewed (approved):
lgtm
